src/libpanic_unwind/Cargo.toml

```diff
@@ -12,6 +12,6 @@ doc = false
 [dependencies]
 alloc = { path = "../liballoc" }
 core = { path = "../libcore" }
-libc = { path = "../rustc/libc_shim" }
+libc = { version = "0.2", default-features = false }
 unwind = { path = "../libunwind" }
-compiler_builtins = { path = "../rustc/compiler_builtins_shim" }
+compiler_builtins = "0.1.0"
```

Comment:
Why don't we have the feature enabled here? Is it because this compiler builtins doesn't actually need it?

Reply:
This depends on `unwind` which depends on the feature so we don't need to activate it twice here.